src/components/challenge/_.module.css

```diff
@@ -0,0 +1,87 @@
+.challenge {
+  background-color: white;
```

Comment:
There is not an opportunity to see a description of the challenge on the screen.
![image](https://user-images.githubusercontent.com/82516098/125385461-3303e600-e3a3-11eb-867f-360881e2ff64.png)